src/components/product/ProductCard.tsx

```diff
@@ -0,0 +1,79 @@
+import styled from "styled-components";
+import { Product } from "../../types/Product";
+import { CardButton } from "./CardButton";
+import { Suspense } from "react";
+
+export const ProductCard = ({ id, name, price, imageUrl }: Product) => {
```

Comment:
```suggestion
export const ProductCard:FC<Product> = ({ id, name, price, imageUrl }) => {
```
위와 같이 FC 타입과 Product 타입을 합하여 명시적으로 선언해줄 수 있을 것 같습니다~